docs/camera_api.rst

```diff
@@ -24,8 +24,14 @@ The previewOff() function will turn off the native camera preview::
 Bebo.Camera.capturePhoto
 ---------------------
     
-`Bebo.Camera.capturePhoto();`
+`Bebo.Camera.capturePhoto(callback);`
 
 The capture() function will capture a photo if the camera preview is open::
 
-    Bebo.Camera.capturePhoto();
\ No newline at end of file
+    Bebo.Camera.capturePhoto(function(err,url){
+    	if (err){
+    		console.error(err);
+    		return;
+    	}
+    	console.log(url); //<--- url is a url to your image.
```

Comment:
should be clarified that it is a local image url that only works on the device. 
if it needs to be displayed everywhere, it should be uploaded to image service.
as we talked about in slack :)